legal-api/src/legal_api/resources/v2/business/business_filings/business_filings.py

```diff
@@ -537,6 +547,7 @@ def process_colin_filing(identifier: str, filing: Filing, business: any) -> Tupl
             current_app.logger.error("Business:%s unable to post to queue, err=%s", identifier, err)
             return {"errors": {"message": "unable to publish for post processing"}}, HTTPStatus.BAD_REQUEST
 
+    # pylint: disable=too-many-statements
```

Comment:
`pylint` run was failing. This is to suppress that.

Reply:
Reverted this change. The two extra if statements are no longer there so pylint is fine now (49/50 statements).